go.mod

```diff
@@ -115,15 +115,15 @@ require (
 	go.opentelemetry.io/otel/trace v1.29.0 // indirect
 	go.uber.org/atomic v1.11.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
-	golang.org/x/crypto v0.26.0 // indirect
+	golang.org/x/crypto v0.31.0 // indirect
```

Comment:
We should separate go mod update to a different PR, it is easier to cherry pick if we need to back patch to easier releases.

Reply:
okay I will remove these changes from this PR